user_story/models/custom_project_task.py

```diff
@@ -1,32 +1,33 @@
-# coding: utf-8
+# -*- coding: utf-8 -*-
+"""
 ##############################################################################
 #
 #    OpenERP, Open Source Management Solution
-#    Copyright (c) 2014 Vauxoo - http://www.vauxoo.com/
+#    Copyright (C) 2004-2009 Tiny SPRL (<http://tiny.be>). All Rights Reserved
```

Comment:
Do not change the copyright, this section is for the copyright of this module itself.